services/brig/test/integration/API/User/Auth.hs

```diff
@@ -456,6 +457,52 @@ testLoginVerify6DigitResendCodeSuccessAndRateLimiting brig galley _opts db = do
         (Just defCookieLabel)
         (Just $ Code.codeValue mostRecentCode)
 
+testLoginVerify6DigitLimitRetries :: Brig -> Galley -> Opts.Opts -> DB.ClientState -> Http ()
+testLoginVerify6DigitLimitRetries brig galley _opts db = do
+  (u, tid) <- createUserWithTeam' brig
+  let Just email = userEmail u
+  let checkLoginFails body =
```

Comment:
There are five occurrences of this function in this file. (One expects a different label.) How about extracting this function and re-using it? :thinking: